radio/src/targets/flysky/hal.h

```diff
@@ -451,6 +451,21 @@ extern void ISR_TIMER3_CAPT_vect(void);
 #define AUX_SERIAL_USART_IRQn             USART1_IRQn
 #define AUX_SERIAL_DMA_Channel_RX         DMA1_Channel3
 
+// AUX2 Serial
+#define AUX2_SERIAL_RCC_AHB1Periph         (RCC_AHBPeriph_GPIOC | RCC_AHBPeriph_DMA1)
+#define AUX2_SERIAL_RCC_APB1Periph         RCC_APB1Periph_USART4
+#define AUX2_SERIAL_GPIO                   GPIOC
+#define AUX2_SERIAL_GPIO_PIN_TX            GPIO_Pin_10 // PC10
+#define AUX2_SERIAL_GPIO_PIN_RX            GPIO_Pin_11 // PC11
+#define AUX2_SERIAL_GPIO_PinSource_TX      GPIO_PinSource10
+#define AUX2_SERIAL_GPIO_PinSource_RX      GPIO_PinSource11
+#define AUX2_SERIAL_GPIO_AF                GPIO_AF_0
+#define AUX2_SERIAL_USART                  USART4
+#define AUX2_SERIAL_USART_IRQHandler       USART4_IRQHandler
+#define AUX2_SERIAL_USART_IRQn             USART4_IRQn
```

Comment:
Same for this line, USART4_IRQn does not exist for F0, it should be USART3_4_IRQn